### DISCUSSION TITLE
ScriptReference/Collider-bounds

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#107] [Notes]
# ScriptReference/Collider-bounds

https://docs.unity3d.com/ScriptReference/Collider-bounds.html

**Comment by docscomments** [comment]
**Comment by Karel Crombecq:**  

Bounds are only updated when a physics tick happens. You can change this behaviour with [`Physics.autoSyncTransforms`](https://docs.unity3d.com/ScriptReference/Physics-autoSyncTransforms.html), or sync manually with [`Physics.SyncTransforms`](https://docs.unity3d.com/ScriptReference/Physics.SyncTransforms.html), but it has a performance overhead. ([source](https://forum.unity.com/threads/psa-collider-bounds-incorrect-until-phyiscs-update.657919/))

**Comment by cjacobwade** [comment]
Note that these bounds are axis aligned, meaning rotating a collider changes its bounds!
